README.md

```diff
@@ -1,128 +1,23 @@
-# WP-PWA Package Development
+# » Frontity - WPorg Connection
 
-## Installation
-
-Clone the main [WP-PWA](https://github.com/wp-pwa/wp-pwa) repository first. Then, inside the `packages` folder, clone this repository.
-
-Finally, use `npm run start:pwa` to start the development environment.
-
-## Changelog
-
-#### 1.6.4
-
-- Convert WpApi to class
-
-#### 1.6.3
-
-- Fix GTM time
-
-#### 1.6.2
-
-- Fix GTM time
-
-#### 1.6.1
-
-- Fix for duplicate entities
-
-#### 1.6.0
-
-- Remove redux
-
-#### 1.5.15
-
-- Fixes goToPreviousContext bug
-- Add license
-
-#### 1.5.14
-
-- Fixes error when media has no sizes
-
-#### 1.5.13
-
-- Populate original in media when none is present
-
-#### 1.5.12
-
-- Add version to JS
-
-#### 1.5.11
-
-- Fix bug with goToPreviousContext
-
-#### 1.5.10
-
-- Populate endpoints without autodiscover
+[![Build Status](https://travis-ci.org/frontity/wp-org-connection.svg?branch=master)](https://travis-ci.org/frontity/wp-org-connection)
 
-#### 1.5.9
+This package connects the [Frontity framework](https://github.com/frontity/frontity) with a self-hosted WordPress.org site.
 
-- Revert AA patch
-
-#### 1.5.8
-
-- Add parameter to override perPage
-- Add headMeta to authors
-
-#### 1.5.7
-
-- Start using wp:contentmedia
-- Add patch until AA updates their purge
-
-#### 1.5.6
-
-- Fix head content being rendered each time
-
-#### 1.5.5
-
-- Last fixes for connection
-
-
-#### 1.5.4
-
-- iOS bug fixes of new connection
-
-#### 1.5.3
-
-- More refactoring and bug fixes of new connection
-
-#### 1.5.2
-
-- Refactor MST implementation
-- Some bug fixes
-
-#### 1.5.1
-
-- Several bug fixes to new connection
-
-#### 1.5.0
-
-- New connection
-
-#### 1.4.1
-
-- Go back to initialUrl and check if it's complete
-
-#### 1.4.0
-
-- Add initialUri compatibility
-- Remove latest from connection until the new connection is here
-
-#### 1.3.2
+## Installation
 
-- Make excerpt optional
+Clone the main [Frontity](https://github.com/frontity/frontity) repository first. Then, inside the `packages` folder, clone this repository.
 
-#### 1.3.1
+Finally, use `npm run start:pwa` to start the development environment.
 
-- Add url in site info for blogs without home on /
-- Fix problem with missing info on media
+## Changelog
```

Comment:
This line is not rendering correctly.

Reply:
It's probably an irregular space.

Reply:
Solved in https://github.com/frontity/wp-org-connection/pull/17/commits/54ad55159425c6b9fc8df00cd553db5d4f5f3506